### ISSUE TITLE
Browserify config requiring Vueify

### ISSUE THREAD

**Issue by fvanwijk** [monterail/vue-multiselect#140]
I want to use the component mixins only, but when I import from `vue-multiselect` my Browserify process complains that `vueify` is missing. I'm not using Vueify (and don't want to) because only the mixins are imported, not the `.vue` file.

It seems that the `browserify` config in package.json is overriding my own browserify config. Is it possible to remove it from package.json and move it a another config file in the project?

**Comment by shentao** [comment]
Damn. It was added there to fix some other problems with Browserify. Would you be able to check if removing it, wouldn’t introduce any problems with Browserify (and Vueify) when someone would want to use the `.vue` file?
If yes, I could remove the additional settings from package.json.

**Comment by fvanwijk** [comment]
I'm quite sure that if anyone wants to use the `.vue` source file, they just have to include the `vueify` transform in their build config themselves.
Alternative is to split the js and html part in two files, or inline the HTML by doing:

```js
template: `
// template here
`
```

I will check this to be sure everything still works by removing the browserify config.

**Comment by shentao** [comment]
This would help me a lot. 
Thanks! 

Let me know what are the results of those checks!

**Comment by fvanwijk** [comment]
I created a sample repo to demonstrate the problem:
https://github.com/fvanwijk/vue-multiselect-browserify-example

TL;DR remove the `browserify` config from `package.json` and add to readme that developers need to add a global `vueify` transform if they want to import the `.vue` files.

**Comment by shentao** [comment]
Done. Should be released soon.
